vgstation13.dme

```diff
@@ -2219,7 +2226,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```

Comment:
Obviously the reason it's included is that he was doing testing

It's good to point out so he doesn't forget to remove it, but you've been around to know 'why'